zetaclient/outboundprocessor/outbound_processor_manager.go

```diff
@@ -0,0 +1,68 @@
+package outboundprocessor
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/rs/zerolog"
+)
+
+type Processor struct {
```

Comment:
maybe rename the source file as `outbound_processor.go` to align with the struct name?

Reply:
It is already in the `outbound_` namespace, so Processor is enough. When initializing you would have `outboundprocessor.Processor`

Also we should rename this, `outbound_processor_manager.go` is a terrible name.
Maybe we can also refactor the code to not use this structure anymore

Reply:
I left it to processor, and called the receiver `p` as it is standard to use first letter of the type for receiver in Go
738fa483